src/components/AddTodoForm/AddTodoForm.jsx

```diff
@@ -0,0 +1,26 @@
+import './AddTodoForm.css';
+import Button from '../Button/Button';
+
+function AddTodoForm({ onAddTodo }) {
+
+	const handleAddTodo = (event) => {
+		event.preventDefault();
+		const todoTitle = event.target.elements.todoTitle.value;
+		console.log(todoTitle);
+		onAddTodo(todoTitle);
+		event.target.reset();
+	};
+
+	return (
+		<form className="addTodoForm" onSubmit={handleAddTodo}>
+			<label htmlFor="todoTitle">Enter ToDo</label>
+			<div className="inputWrapper">
+				<input type="text" id="todoTitle"/>
+				<Button />
```

Comment:
NON-BLOCKING: If you wanted to make this a more generalizable component that could be used in multiple places in your app, you could add props such as - `className`, `label`, and `type` so that you can use the button in a variety of places.